asmcomp/amd64/emit.mlp

```diff
@@ -171,6 +171,13 @@ let emit_Llabel fallthrough lbl =
   if not fallthrough && !fastcode_flag then D.align 4;
   def_label lbl
 
+let load_label_addr s arg =
+  (* CR mshinwell: this needs more testing *)
```

Comment:
Is this patch supposed to still have CRs in it?

Reply:
This still needs to be checked on the various targets (and target variants) to make sure that the correct instruction sequence is being generated.